pkg/gce-pd-csi-driver/cache.go

```diff
@@ -563,7 +563,7 @@ func isCachingSetup(mainLvName string) (error, bool) {
 func fetchChunkSizeKiB(cacheSize string) (string, error) {
 	var chunkSize float64
 
-	cacheSizeInt, err := common.ConvertGiStringToInt64(cacheSize)
+	cacheSizeInt, err := strconv.ParseInt(cacheSize, 10, 64)
```

Comment:
The comment below needs to be updated too? "Chunksize should be divisible by 32Kib so we need (chunksize/32*1024)*32*1024"?